### ISSUE TITLE
[BUG] TEST_RUNNER_PATH not taken into account

### ISSUE THREAD

**Issue by Zenor27** [callstack/reassure#324]
**Describe the bug**
When specifying TEST_RUNNER_PATH, the specified path is not taken into account, the default path `node_modules/.bin/jest` is used.

I think the issue is in the measure command (`measure.ts`):

The current line that grabs the test runner path:

```
  const testRunnerPath =
    process.env.TEST_RUNNER_PATH ?? process.platform === 'win32'
      ? 'node_modules/jest/bin/jest'
      : 'node_modules/.bin/jest';
```

Should be:

```
  const testRunnerPath =
    process.env.TEST_RUNNER_PATH ?? (process.platform === 'win32'
      ? 'node_modules/jest/bin/jest'
      : 'node_modules/.bin/jest');
```

**To Reproduce**
1. Launch `TEST_RUNNER_PATH=somepath yarn reassure`
2. Node throw an error:  `node_modules/.bin/jest`Cannot be found

**Expected behavior**

The TEST_RUNNER_PATH environment variable should be used instead of the default.

**Desktop:**
 - OS: macOS
 - Version 13.1

**Comment by mdjastrzebski** [comment]
@Zenor27 good catch will release 0.7.1 today.

**Comment by mdjastrzebski** [comment]
Released in version 0.7.1.
